mt-kahypar/partition/refinement/fm/strategies/local_gain_cache_strategy.h

```diff
@@ -203,7 +201,8 @@ class LocalGainCacheStrategy {
   std::pair<PartitionID, HyperedgeWeight> computeBestTargetBlock(const PartitionedHypergraph& phg,
                                                                  const GainCache& gain_cache,
                                                                  const HypernodeID u,
-                                                                 const PartitionID from) {
+                                                                 const PartitionID from,
+                                                                 bool ignore_balance) {
```

Comment:
Do you think this would be faster if ```ignore_balance``` was a template parameter.

Reply:
No. The function is inlined (marked as always_inline) and the parameter is passed as a constant literal. It is thus trivial for the compiler to do the same optimizations as with a template parameter